.github/workflows/linters.yml

```diff
@@ -0,0 +1,63 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
```

Comment:
- [ ] As requested previously please ensure GitHub actions are running and are all green before requesting a re-review.